tomviz/ResetReaction.cxx

```diff
@@ -30,6 +33,16 @@ void ResetReaction::updateEnableState()
 
 void ResetReaction::reset()
 {
+  if (ModuleManager::instance().hasDataSources()) {
+    if (QMessageBox::Yes !=
+        QMessageBox::warning(tomviz::mainWidget(), "Reset Warning",
+                             "Current data and operators will be cleared when "
+                             "resetting.  Proceed anyway?",
```

Comment:
Just one space after a period, I think this could be shortened to "Data may be lost when resetting. Are you sure you want to reset?" I think the title can be shortened to "Reset" too.